cli/dts/lib.deno.shared_globals.d.ts

```diff
@@ -712,7 +721,26 @@ declare class Worker extends EventTarget {
        * hello world2
        *
        */
-      deno?: boolean;
+      deno?: {
+        namespace?: boolean;
+        permissions?: "inherit" | {
+          env?: "inherit" | boolean;
+          hrtime?: "inherit" | boolean;
+          /**
+           * The format of the net access list must be `hostname[:port]`
+           * in order to be resolved
```

Comment:
How do you go about creating worker that has no permissions at all? Could you include it as example in the doc string?

Reply:
I just added `false` as an option for the `permissions` prop. This disables all of the worker access. I'll add docs to the manual soon